community/technical-team.md

```diff
@@ -37,12 +37,12 @@ The schedule for these developer workshops is the _last Friday of every odd mont
 
 **2024**
 
-* November 29, 2024 [4pm UTC](https://www.timeanddate.com/worldclock/converter.html?iso=20241129T160000&p1=1440&p2=994&p3=43) ODD Friday
-* October 31, 2024 [4pm UTC](https://www.timeanddate.com/worldclock/converter.html?iso=20241031T160000&p1=1440&p2=994&p3=43) ODD Thursday
-* September 27, 2024 [3pm UTC](https://www.timeanddate.com/worldclock/converter.html?iso=20240927T150000&p1=1440&p2=994&p3=43) ODD Friday
+* November 29, 2024 [1pm UTC](https://www.timeanddate.com/worldclock/converter.html?iso=20241129T130000&p1=1440&p2=994&p3=43) ODD Friday
+* October 31, 2024 [1pm UTC](https://www.timeanddate.com/worldclock/converter.html?iso=20241031T130000&p1=1440&p2=994&p3=43) ODD Thursday
+* September 27, 2024 [12am UTC](https://www.timeanddate.com/worldclock/converter.html?iso=20240927T120000&p1=1440&p2=994&p3=43) ODD Friday
```

Comment:
Are you sure we should meet at midnight?

Reply:
But you're right. To avoid further confusion, should we switch to 12:00 UTC instead?

Reply:
This could also confuse people. We should switch to 24 hours for all dates. See https://www.npl.co.uk/resources/q-a/is-midnight-12am-or-12pm

Reply:
Switched all dates to 24 hour system